indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -109,7 +109,9 @@ export default function FormItem({
   const InputComponent = meta.inputComponent;
   const inputProps = {title, description, isEnabled, fieldId: id, ...rest};
   const showPurged = !setupMode && isPurged;
-  const disabled = !isEnabled || showPurged || !!lockedReason || (paidItemLocked && !isManagement);
+  const isReadOnly = inputType === 'email' && rest.readonlyEmail;
```

Comment:
this would affect all email inputs, not just the one in the personal data section